docs/installation/linux/debian.md

```diff
@@ -36,6 +36,26 @@ Docker is supported on the following versions of Debian:
 
      $ uname -r
 
+ Additionally, for users of Debian Wheezy, backports must be available. To enable backports in Wheezy:
+ 
+ 1. Log into your machine and open a terminal with `sudo` or `root` privileges.
+ 
+ 2. Open the `/etc/apt/sources.list.d/backports.list` file in your favorite editor.
+
+     If the file doesn't exist, create it.
+ 
+ 3. Remove any existing entries.
+ 
+ 4. Add an entry for backports on Debian Wheezy.
+
+     An example entry:
+
+    deb http://http.debian.net/debian wheezy-backports main
+
+ 5. Update package information:
+
+         $ apt-get update
```

Comment:
@deed02392 Oh, you're right, it's only the last steps (below "install docker") that are using `sudo`. Perhaps we should add a "1. If you haven't already..." to that block.

It's difficult finding the right balance here; for some people it's "obvious" you have be privileged, or use `sudo`. For others, we keep getting reports "it doesn't work!".

Okay, let's keep out the `sudo` here; I'll do a follow up PR to make the last block consistent 👍